docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -115,82 +119,133 @@ POST /transactions/
         "version": 1
       }
 
+   :resheader Content-Type: ``application/json``
+
+   :statuscode 200: A transaction with that ID was found.
+   :statuscode 404: A transaction with that ID was not found.
+
+.. http:get:: /transactions
+
+   The current ``/transactions`` endpoint returns a ``404 Not Found`` HTTP
+   status code. Eventually, this endpoint might get implemented, its usefulness
+   would be questionable however, as the transactions listed would change by
+   the second.
+
+   There are however requests that might come of use, given the endpoint is
+   queried correctly. Some of them include retrieving a list of transactions
+   that include:
+
+   * `Unfulfilled conditions <#get--transactions?fields=id,conditions&fulfilled=false&owner_afters=owners_after>`_
+   * `A specific asset <#get--transactions?fields=id,asset,operation&operation=CREATE|TRANSFER&asset_id=asset_id>`_
+   * `Specific metadata <#get--transactions?fields=id,metadata&metadata_id=metadata_id>`_
+
+   In this section, we've listed those particular requests, as they will likely
+   to be very handy when implementing your application on top of BigchainDB.
+   A generalization of those parameters can follows:
+
+   :query fields: A comma separated string to expand properties on the transaction object to be returned.
+   :type fields: string
+
+   :query fulfilled: A flag to indicate if transaction's with fulfilled conditions should be returned.
+   :type fulfilled: boolean
+
+   :query owners_after: Public keys able to validly spend an output of a transaction, assuming the user also has the corresponding private key.
+   :type owners_after: base58 encoded string
+
+   :query operation: One of the three supported operations of a transaction.
+   :type operation: string
+
+   :query asset_id: asset ID.
+   :type asset_id: uuidv4
+
+   :query metadata_id: metadata ID.
+   :type metadata_id: uuidv4
+
+
+.. http:get:: /transactions?fields=id,conditions&fulfilled=false&owner_afters={owners_after}
```

Comment:
It might be more pleasant for the user to use 
``GET /transactions/unfulfilled?fields=id,conditions&owners_after={owners_after}``
opposed to/or in conjunction with
``GET /transactions/?fields=id,conditions&fulfilled=false&owners_after={owners_after}``

this way it's more clear that ``GET /transactions`` is a different endpoint - and this one has actually better mapping to what people in the blockchain space call unspent outputs (UTXO)

from http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api
## Aliases for common queries
To make the API experience more pleasant for the average consumer, consider packaging up sets of conditions into easily accessible RESTful paths. For example, the recently closed tickets query above could be packaged up as GET /tickets/recently_closed